mitodl/mitxonline
# Nl/nocache staf dash

#### Pre-Flight checklist


#### What are the relevant tickets?
N/A

#### What's this PR do?
I discovered during RC testing that fastly was caching the refine app view, so I added `never_cache` to that view and also the user dashboard view for good measure.

#### How should this be manually tested?
Go to both `/staff-dashboard` and `/dashboard` and verify you see the header `Cache-Control | max-age=0, no-cache, no-store, must-revalidate, private` in the response.
